docs/admin/index.md

```diff
@@ -12,13 +12,11 @@ infrastructure which makes the project possible.
    ../README
    /tools/wptrunner/README.rst
    /tools/wptserve/docs/index.rst
-   pywebsocket3
 
 .. toctree::
    :hidden:
 
    ../tools/wptserve/README
-   ../tools/third_party/pywebsocket3/README
```

Comment:
Just double checking these changes are intentional? There's still a README, but it was renamed.

Reply:
Hm, I should have asked but I thought that we had this entry because it was a custom tool before. But now it's used like any other Python package under `third-party` from PyPI. Maybe my assumption was wrong and I should restore that line? What are these entries in that file really used for?

Reply:
I have no idea why this was here, if you've looked at the resulting docs before and after and think it's an improvement that's fine with me.

Reply:
@jgraham do you maybe have an idea, and if we still need to mention pywebsocket3 here?

Reply:
Let me just revert these changes. When it's clear that we don't need it we can remove it in a separate PR.